release.sh

```diff
@@ -11,9 +11,18 @@ if ! env | grep GITHUB_TOKEN= > /dev/null; then
   exit 1
 fi
 
+# Run git fetch to ensure that the origin is updated.
+git fetch
+
+# Ensure that the maint branch is included in this release.
+if ! git merge-base --is-ancestor origin/maint HEAD; then
+  echo "maint branch has not been merged into $(git rev-parse --abbrev-ref HEAD)." 2>&1
+  exit 1
+fi
+
 export GO111MODULE=on
 
 version=$(go run ./internal/cmd/changelog nextver)
 git tag -s -m "Release $version" $version
-git push origin $version
+git push origin "$version" "HEAD:maint"
```

Comment:
What does this line do? Is it just moving the `main` branch to the latest commit?

Reply:
Yea. It's moving whatever the `HEAD` ref is (which is what you just tagged) and moving `maint` to point to that, but without having to modify anything locally.

Unfortunately, you can't put the tag in there directly so `$version:maint` doesn't work. I tried it out in a test repository before committing this.